NetKVM/wlh/ParaNdis6-Oid.cpp

```diff
@@ -703,8 +703,8 @@ void ParaNdis6_FillOffloadConfiguration(PARANDIS_ADAPTER *pContext)
     NDIS_OFFLOAD *po = &pContext->ReportedOffloadConfiguration;
     FillOffloadStructure(po, pContext->Offload.flags);
 #if PARANDIS_SUPPORT_RSC
-    po->Rsc.IPv4.Enabled = (pContext->RSC.bIPv4SupportedSW && pContext->RSC.bIPv4SupportedHW);
```

Comment:
Hi,

Thank you for the patch.

In general - if backend doesn't support RSC and we set this feature, the driver will still function.
But I don't think it is a good idea. Also this patch might hide several actual issues (OnSetRSCParameters overrides initial setting of RSC.bIPv4Enabled and  RSC.bIPv6Enabled).

Best regards,
Yan.